app/locale/en.coffee

```diff
@@ -614,19 +614,10 @@
 
   subscribe:
     subscribe_modal_title: "CodeCombat Premium"
-    comparison_blurb: "Sharpen your skills with a <strong>CodeCombat Premium</strong> subscription!"
+    comparison_blurb: "Become a Master Coder - subscribe to <b>Premium</b> today!" # {change}
```

Comment:
I'm not sure what you mean by "correct tag". The "change" string is to indicate to translators that they need to update their strings for this.